### PR TITLE
Social Share Buttons

### LINKED ISSUE #856: Social Share Buttons
People should be able to share/tweet/like content on the site.

Initiative #845

### PR DESCRIPTION
**Pull request by pdcp1**
This PR:
- Adds a new component `SocialShareButtons` with Twitter, LinkedIn, Email, and Facebook share buttons
- Adds share buttons to incident page
- Adds share buttons to blog post page
- Adds a link to the incident to the Submission Review toast message, so the user can click and see the incident page and share it.

**This is the toast for a report associated with an existing incident:**

Screen Shot 2022-08-03 at 21 41 19

**This is the toast for a report just approved that creates a new incident:**

Screen Shot 2022-08-03 at 21 40 21

### Incident share buttons

image

### Blog post share buttons

image

Closes #856

### CODE REVIEW

**Comment by cesarvarela** [comment]
Should I get something elese than this?
image

seems to be missing the open graph stuff?

**Comment by pdcp1** [comment]
@cesarvarela I'm not sure if the openGraph improvements are already merged to Production. That's why the share preview in your case is not working.

**Comment by cesarvarela** [comment]
Looks like it is working for fb
image

**Comment by cesarvarela** [comment]
yes the open graph improvements are live

**Comment by cesarvarela** [comment]
please fix conflicts!

**Comment by pdcp1** [comment]
Conflicts resolved!

**Comment by cesarvarela** [comment]
The twitter test is failing

**Comment by pdcp1** [comment]
Twitter test fixed

**Review by cesarvarela — CHANGES_REQUESTED** [review]
Looks goods, but it is missing translations.

**Comment by pdcp1** [comment]
@cesarvarela Translations added

File: site/gatsby-site/src/components/submissions/SubmissionReview.js [hunk 1]
```diff
@@ -254,7 +268,7 @@ const SubmissionReview = ({ submission }) => {
               disabled={!isSubmitter || promoting}
               onClick={promoteSubmission}
             >
-              {submission.incident_id === 0 ? <>Add New Incident</> : <>Add New Report</>}
+              {isNewIncident ? <>Add New Incident</> : <>Add New Report</>}
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/submissions/SubmissionReview.js:271]: Missing `Trans`

File: site/gatsby-site/src/components/submissions/SubmissionReview.js [hunk 2]
```diff
@@ -271,7 +285,7 @@ const SubmissionReview = ({ submission }) => {
               disabled={!isSubmitter || deleting}
               onClick={rejectReport}
             >
-              {submission.incident_id === 0 ? <>Reject New Incident</> : <>Reject New Report</>}
+              {isNewIncident ? <>Reject New Incident</> : <>Reject New Report</>}
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/submissions/SubmissionReview.js:288]: ditto

File: site/gatsby-site/src/components/ui/SocialShareButtons.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            'width=600,height=300'
+          );
+          return false;
+        }}
+      >
+        <FontAwesomeIcon
+          icon={faEnvelope}
+          color={'#001934'}
+          className={'pointer fa fa-lg'}
+          title="Share by email"
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/ui/SocialShareButtons.js:68]: ditto

File: site/gatsby-site/src/components/ui/SocialShareButtons.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            'width=580,height=296'
+          );
+          return false;
+        }}
+      >
+        <FontAwesomeIcon
+          icon={faFacebook}
+          color={'#001934'}
+          className={'pointer fa fa-lg'}
+          title="Share to Facebook"
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/ui/SocialShareButtons.js:89]: ditto

**Review by cesarvarela — CHANGES_REQUESTED** [review]

**Review by cesarvarela — APPROVED** [review]

**Pull request merged**